src/private/meetingRoom.ts

```diff
@@ -0,0 +1,216 @@
+import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
+import { GlobalVars } from '../internal/globalVars';
+import { SdkError } from '../public/interfaces';
+
+export namespace meetingRoom {
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to represent a meeting room.
+   */
+  export interface MeetingRoomInfo {
+    /**
+     * Endpoint id of the meeting room.
+     */
+    endpointId: string;
+    /**
+     * Device name of the meeting room.
+     */
+    deviceName: string;
+    /**
+     * Client type of the meeting room.
+     */
+    clientType: string;
+    /**
+     * Client version of the meeting room.
+     */
+    clientVersion: string;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Enum used to indicate meeting room capabilities.
+   */
+  export enum Capability {
+    /**
+     * Media control capability: toggle mute.
+     */
+    ToggleMute = 'toggleMute',
```

Comment:
Please make these match exactly. There is no reason for them to not be the exact same and in the future not having them be the same could easily cause bugs when passing these enum values between apps. For example:
```
ToggleMute = 'ToggleMute',
ToggleCamera = 'ToggleCamera'
```

Reply:
Thanks Yuri. Has updated the enum declarations to match their values. Since the values can not be changed as they are a part of commands protocol.